docs/classification/workflow.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+parent: Classification
+title: Pigeon CLI Workflow
+nav_order: 3
+---
+
+## CLI Workflow
+
+### Map reads to a reference genome
+
+Map reads using _pbmm2_.
+
+```
+pbmm2 align --preset ISOSEQ --sort <input.bam> <ref.fa> <mapped.bam>
+```
+
+### Collapse transcripts into unique isoforms
+
+Collapse mapped reads into unique isoforms using _isoseq collapse_.
+
+```
+isoseq3 collapse <mapped.bam> <collapse.gff>
+```
+
+### Sort input transcript GFF
+
+Sort the transcript GFF file output from _isoseq collapse_.
+
+```
+pigeon sort <collapse.gff> -o sorted.gff
+```
+
+### Index the genome annotation
+
+Index the genome annotation file before classification.
```

Comment:
I would mention that the supplemental annotation files (CAGE & intropolis) need to be pigeon-indexed as well.